_data/risc_outgoing.yml

```
@@ -33,6 +33,75 @@
     }
   }
 
+- friendly_name: Account Disabled
+  direction: outgoing
+  event_type: https://schemas.openid.net/secevent/risc/event-type/account-disabled
+  spec_url: "https://openid.net/specs/openid-risc-event-types-1_0-ID1.html#account-disabled"
+  description: >
+    Login.gov pushes this event when a user account suspended.
+  payload_schema: &iss_sub_schema
```

I think we can drop the `&iss_sub_schema` here if we're not reusing this reference
```suggestion
  payload_schema:
```